Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/MarkerDetection/ArUco/ArUcoMarkerDetectorSpatialLocalizer.cs

```diff
@@ -11,25 +12,33 @@
 namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
 {
     /// <summary>
-    /// Helper spatial localization method where the host finds a coordinate and gives its id to the observer
+    /// SpatialLocalizer that detects an ArUco marker
     /// </summary>
-    internal abstract class BroadcasterCoordinateSpatialLocalizationMechanism : SpatialLocalizationMechanismBase
+    internal class ArUcoMarkerDetectorSpatialLocalizer : SpatialLocalizer
```

Comment:
This class was supposed to be generic, I was going to extend it for ASA

Reply:
I will push changes to split back this functionality for ASA and will keep ArUcoMarkerVisualizerSpatialLocalizer still inheriting from SpatialLocalizer.